File: services/dashboard-service/src/main/java/com/itjob/dashboardservice/controller/HealthController.java
package com.itjob.dashboardservice.controller;

import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

import java.time.Instant;
import java.util.Map;

@RestController
public class HealthController {

    @GetMapping("/health")
    public Map<String, Object> getHealth() {
        return Map.of(
                "status", "ok",
                "service", "dashboard-service",
                "timestamp", Instant.now().toString()
        );
    }
}

File: services/organization-service/src/health.controller.ts
import { Public } from '@/common/decorators';
import { Controller, Get } from '@nestjs/common';

@Controller('health')
export class HealthController {
  @Get()
  @Public()
  getHealth() {
    return {
      status: 'ok',
      service: 'organization-service',
      timestamp: new Date().toISOString(),
    };
  }
}
